SCHEMA

```diff
@@ -0,0 +1,72 @@
+Canonical Poke datbase information
+Current as of db_version '0' (initial production version of the db)
+
+== Database Schema ==
```

Comment:
Leaving canonical-poke stuff in place as-is at the moment, pending decision on how/if we're going to do anonymous machine counting.